web/yo/app/scripts/controllers/gene.js

```diff
@@ -2514,17 +2514,78 @@ angular.module('oncokbApp')
                 mutationEffect.addOn.setText('');
             };
 
-            $scope.move = function(driveList, index, moveIndex, event) {
+            $scope.displayAngle = function (parent, item, type) {
+                var index = parent.indexOf(item);
+                switch (type) {
+                case 'top':
+                    if (index === 0) {
+                        return false;
+                    } else {
+                        return true;
+                    }
+                case 'bottom':
+                    if (index === parent.length - 1) {
```

Comment:
The original logic is `index < parent.length - 2`

Reply:
Current implementations are equivalent to original logic.